h/templates/admin/users.html.jinja2

```diff
@@ -43,8 +43,34 @@
           </tr>
         </tbody>
       </table>
+
+      <form id="users-delete-form" method="POST" action="{{request.route_path('admin_users_delete')}}" class="form-inline">
+        <input type="hidden" name="username" value="{{user.username}}">
+
+        {% if user_meta['annotations_count'] > 100 %}
```

Comment:
This is quite an arbitrarily set number, since I'm going through the `api.storage` module, deleting each annotation results in two requests to Elasticsearch.